internal/identity_resolution/engine/scorer.go

```
@@ -0,0 +1,163 @@
+
+		val2 := candidate.GetAttribute(rule.PropertyName)
+		if val2 == "" {
+			logger.Debug(fmt.Sprintf("Scorer: rule '%s' — candidate missing value (score=0, weight=%.0f)",
+				rule.RuleName, weight))
+			// Counts toward applicable weight but contributes 0 to the sum.
+			continue
+		}
+
+		// todo: should I remove mode?
+		effectiveMode := mode
+		if rule.UnificationMethod == "fuzzy" {
```

golangci-lint (2.11.4)

[error] QF1003: could use tagged switch on rule.UnificationMethod

(staticcheck)